docs/source/users/index.md

```diff
@@ -93,10 +93,13 @@ and create an environment that uses Python 3.11:
 
     $ conda create -n jupyter-ai python=3.11
     $ conda activate jupyter-ai
-    $ pip install jupyter_ai
 
-Then, follow the steps from "Requirements" and "Installation via `pip`" to
-install JupyterLab and Jupyter AI in this Conda environment.
+Then, use either `pip` or `conda` to install JupyterLab and Jupyter AI in this
+Conda environment.
+
+    $ pip install jupyter-ai                   # or,
+    $ conda install -c conda-forge jupyter-ai  # or,
+    $ conda install conda-forge::jupyter-ai
```

Comment:
So when you install a package in a conda environment using pip _very bad things_ can happen. They usually don't if you have `pip` installed explicitly in the conda environment, but if it is inherited from system installation and your package depends on anything more complex (like jupyter stack or compiled code - both applicable here), a fraction of users will find themselves questioning their sanity with a problem rather hard to debug.

```suggestion
Then, use `conda` to install JupyterLab and Jupyter AI in this Conda environment.

    $ conda install -c conda-forge jupyter-ai  # or,
    $ conda install conda-forge::jupyter-ai
```